rom-or556.c

```diff
@@ -0,0 +1,52 @@
+//------------------------------------------------------------------------------
+// EMU86 - Orkitt 556 custom ROM
+//------------------------------------------------------------------------------
+
+#include "emu-int.h"
+#include "emu-mem-io.h"
+
+#include <stdlib.h>
+
+
+// Interrupt handler table
+
+int_num_hand_t _int_tab [] = {
+//	{ 0x03, int_03h },
+	{ 0,    NULL    }
+	};
+
+
+// ROM stub initialization
+
+void rom_init (void)
+	{
+	// ROM boot start @ F000:FF00h
+	// ELKS kernel @ E000:0h
+
+	mem_stat [0xFFF00] = 0x9A;  // CALLF E000:0003h
+	mem_stat [0xFFF01] = 0x03;
+	mem_stat [0xFFF02] = 0x00;
+	mem_stat [0xFFF03] = 0x00;
+	mem_stat [0xFFF04] = 0xE0;
+
+	mem_stat [0xFFF05] = 0xCD;  // INT 19h
+	mem_stat [0xFFF06] = 0x19;
+
```

Comment:
Also curious here: using CALLF E000:0003h followed by INT 19h has advantage of using fixed address (option ROM start+3), known entry mechanism into ELKS (or other option-ROM compatible OS). Are you thinking this should become permanent standard entry mechanism for ELKS? That would also possible movement of _startos entry point without embedded INIT ROM changes. If so, will document in ELKS.

Reply:
As using the BIOS extension protocol obviously costs almost nothing (2 instructions), I would stay on that ABI for now.